src/vs/workbench/services/configuration/common/configurationEditing.ts

```diff
@@ -210,7 +210,7 @@ export class ConfigurationEditing {
 			if ((<FileOperationError>error).fileOperationResult === FileOperationResult.FILE_MODIFIED_SINCE) {
 				throw this.toConfigurationEditingError(ConfigurationEditingErrorCode.ERROR_CONFIGURATION_FILE_MODIFIED_SINCE, operation.target, operation);
 			}
-			throw this.toConfigurationEditingError(ConfigurationEditingErrorCode.ERROR_INTERNAL, operation.target, operation);
+			throw new ConfigurationEditingError(nls.localize('fsError', "Error while writing to {0}. {1}", this.stringifyTarget(operation.target), error.message), ConfigurationEditingErrorCode.ERROR_INTERNAL);
```

Comment:
Just making sure this isn't sent to telemetry, if it is we would get one per language per operation target per error message

Reply:
> if it is we would get one per language per operation target per error message

Not sure if I understood this. Why is it one per... ?

Reply:
Ah got it. This caller of this can be API or user. Since user can see this error, it is translated.